ompi/communicator/comm.c

```diff
@@ -2032,3 +2404,28 @@ static int ompi_comm_copy_topo(ompi_communicator_t *oldcomm,
     newcomm->c_flags |= newcomm->c_topo->type;
     return OMPI_SUCCESS;
 }
+
+char *ompi_comm_print_cid (const ompi_communicator_t *comm)
```

Comment:
This thread-local 2 correct values function begs for documentation !

Reply:
Indeed. I thought I documented this somewhere.... Guess not. Should add documentation to the header declaration.